test/unit/ecto_paging_test.exs

```diff
@@ -184,6 +184,18 @@ defmodule Ecto.PagingTest do
       assert paging.has_more
     end
 
+    test "ending_before is not nil" do
+      {[head | _] = items, paging} = Ecto.Paging.TestRepo.page(Ecto.Paging.TestSchema, %{limit: 150})
+      assert 150 == Enum.count(items)
+      assert %{cursors: %{ending_before: ending_before}, has_more: false} = paging
+      assert head.id == ending_before
+    end
+
+    test "first page has not ending_before" do
```

Comment:
The first page should have ending_before so that you can fetch newer items when they arrive.